app/helpers/sprangular/application_helper.rb

```diff
@@ -16,7 +16,8 @@ def js_environment
         locale: I18n.locale,
         currency: Money::Currency.table[current_currency.downcase.to_sym],
         translations: current_translations,
-        templates: template_paths
+        templates: template_paths,
+        api_domain: Spree::Config.api_domain || ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.